gui/src/components/logs/LocalLogProvider.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
```

Comment:
is there a notion of a RemoteLogProvider? Seems like this could be called "LogProvider", no?